docstring_parser/rest.py

```diff
@@ -38,12 +38,16 @@ def _build_meta(args: T.List[str], desc: str) -> DocstringMeta:
                 f"Expected one or two arguments for a {key} keyword."
             )
 
+        m = re.match(r'.*defaults to (.+)', desc)
```

Comment:
This will also capture commas, ending periods etc :unamused: I'd add negative lookahead to prevent this and some tests, also I'd support docstrings of the form `timeout (default: 5)`

Reply:
I'm on it

Reply:
The negative lookahead is not so straightforward.

All these are valid

```python
# Strings can contain almost anything
defaults to 'Hello, World!'
# Floats contain punctuation
defaults to 5.0
# Floats imply ".0" - is this punctuation or a float?
defaults to 1.
```

Reply:
Where can I find examples of the `timeout (default: 5)' style?

Reply:
Some of the closed source projects I work on :grin:
What do you suggest? Maybe let's leave it as-is? I mean it's still better than nothing

Reply:
I think it's a starting point. It won't break anything.

Reply:
Cool, let me merge this then and thank you